test/trento/application/projectors/cluster_projector_test.exs

```diff
@@ -7,14 +7,20 @@ defmodule Trento.ClusterProjectorTest do
 
   import Trento.Factory
 
-  alias Trento.Domain.Events.{
+  alias Trento.Clusters.ValueObjects.{
```

Comment:
This file must be moved to the correct folder (`test/trento/clusters/projections/cluster_projector_test.exs`, and change the `defmodule`